Dockerfile

```diff
@@ -18,9 +18,9 @@ RUN sed -e "/deb-src/d" -i /etc/apt/sources.list \
     && rm -rf /.root/cache \
     && rm -rf /var/lib/apt/lists/*
 
-ADD https://dl.google.com/go/go1.15.2.linux-amd64.tar.gz ./go.tar.gz
+ADD https://dl.google.com/go/go1.19.5.linux-amd64.tar.gz ./go.tar.gz
```

Comment:
@jasonparekh thanks for the PR. While on this, why not upgrading to the last version (1.20.1 atm)?